openapi-v1.yaml

```diff
@@ -6323,13 +6328,13 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /taskgraphs/logs:
+  /taskgraphs/{namespace}/logs:
```

Comment:
This endpoint needs to be outside the namespace level; we want users to have an endpoint that lets them see all the logs they have access to (like the way you can see all arraytasks).

Reply:
We already post at a namespace, I see no reason to not do the same for fetch all to start with at least

Reply:
Per our discussion just now, we are OK with a user only being able to retrieve tasks associated with a single namespace for the time being as a product decision. This is different from how we treat ArrayTasks but is more consistent with our other resource types.

Reply:
Posted in open story regarding the endpoints https://app.shortcut.com/tiledb-inc/story/13280/implement-endpoints-for-task-graph-log-reporting-api to pin requirements